db/schema.rb

```diff
@@ -357,15 +367,15 @@
 
   create_table "other_assets_declarations", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.uuid "legal_aid_application_id", null: false
-    t.decimal "second_home_value", precision: 14, scale: 2
-    t.decimal "second_home_mortgage", precision: 14, scale: 2
-    t.decimal "second_home_percentage", precision: 14, scale: 2
-    t.decimal "timeshare_property_value", precision: 14, scale: 2
-    t.decimal "land_value", precision: 14, scale: 2
-    t.decimal "valuable_items_value", precision: 14, scale: 2
-    t.decimal "money_assets_value", precision: 14, scale: 2
-    t.decimal "money_owed_value", precision: 14, scale: 2
-    t.decimal "trust_value", precision: 14, scale: 2
+    t.decimal "second_home_value"
+    t.decimal "second_home_mortgage"
+    t.decimal "second_home_percentage"
+    t.decimal "timeshare_property_value"
+    t.decimal "land_value"
+    t.decimal "valuable_items_value"
+    t.decimal "money_assets_value"
+    t.decimal "money_owed_value"
+    t.decimal "trust_value"
```

Comment:
can we not modify the schema.rb for things unrelated to the PR?
when on the master branch use the `bin/rails db:schema:load` command to align your local DB with the `schema.rb` file in master

Reply:
👍